spec/fixtures/state_file/fed_return_xmls/2023/az/richard_retirement_1099r.xml

```diff
@@ -114,7 +114,7 @@
         <F1099RStateTaxGrp>
           <StateTaxWithheldAmt>11</StateTaxWithheldAmt>
           <StateAbbreviationCd>NC</StateAbbreviationCd>
-          <PayerStateIdNumber>111111111</PayerStateIdNumber>
+          <PayerStateIdNum>111111111</PayerStateIdNum>
```

Comment:
I see in the federal schema that it uses `PayerStateIdNum`... but the xml with `PayerStateIdNumber` was provided to us from the DF team. This change makes sense to me, I'm just confused about how it happened.

Reply:
Resolved by looking w/ martha at the most recent sample XML from kiteworks & observing that it uses the element name from the federal XML schema (PayerStateIdNum)